Fastfile

```diff
@@ -100,6 +100,10 @@ def _build_number
   end
 end
 
+def _get_project_build_number
+  return ENV.include?('TAB_TARGET_FOR_BUILD_NUMBER') ? get_version_number(target: ENV['TAB_TARGET_FOR_BUILD_NUMBER']) : get_version_number
```

Comment:
I can see RuboCop complaining about the length of this line, could you spread it across a couple of lines?

Reply:
@zacoid55  should I do a classic `if else` ? Sorry not a ruby pro at all